TablePlus/TablePlus.jss.recipe

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Description</key>
+	<string>Downloads the latest version of TablePlus and creates a package. Then, uploads to the Jamf Pro Server.</string>
+	<key>Identifier</key>
+	<string>com.github.jss-recipes.jss.TablePlus</string>
+	<key>Input</key>
+	<dict>
+		<key>CATEGORY</key>
+		<string>Productivity</string>
+		<key>GROUP_NAME</key>
+		<string>%NAME%-update-smart</string>
+		<key>GROUP_TEMPLATE</key>
+		<string>SmartGroupTemplate.xml</string>
+		<key>NAME</key>
+		<string>TablePlus</string>
+		<key>POLICY_CATEGORY</key>
+		<string>Testing</string>
+		<key>POLICY_TEMPLATE</key>
+		<string>PolicyTemplate.xml</string>
+		<key>SELF_SERVICE_DESCRIPTION</key>
+		<string>TablePlus update</string>
```

Comment:
I'll merge this in, but I'm going to change it to something more descriptive. Typically the description would describe the purpose of the app.

Reply:
Fixed in: https://github.com/autopkg/jss-recipes/pull/220